draft-ietf-quic-tls.md

```diff
@@ -804,17 +803,12 @@ modifying the contents of packets from future versions.
 The HKDF-Expand-Label function defined in TLS 1.3 MUST be used for Initial
 packets even where the TLS versions offered do not include TLS 1.3.
 
+The secrets used for protecting Initial packets do not change during the
+connection, even after receiving a Retry.
```

Comment:
Is it worth adding some advice on how to implement this?  It's not entirely obvious.

```suggestion
connection, even after receiving a Retry.  A server that sends a Retry
therefore needs to either remember Initial protection keys or
save them in the Retry token.
```